openid-4-verifiable-presentations-1_0.md

```diff
@@ -2404,6 +2407,17 @@ The following is another non-normative example of a W3C Verifiable Credential th
 }
 ```
 
+The following is another non-normative example of a W3C Verifiable Credential that would match the `type_values` DCQL query above (other claims omitted for readability):
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1"
+  ],
+  "type": ["VerifiableCredential", "IdentityCredential"]
```

Comment:
In this case, the "IdentityCredential" is silently dropped by JSON-LD Processors, such as the one used on json-ld playground:

https://json-ld.org/playground/

```
_:b0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://www.w3.org/2018/credentials#VerifiableCredential> .
```

Try pasting the JSON-LD above in, and note the errors and silent dropping when changing tabs.